Packs/Base/Scripts/DBotSuggestClassifierMapping/DBotSuggestClassifierMapping.py

```diff
@@ -526,8 +533,25 @@ def get_alias_index(field_name, alias):
     return SIEM_FIELDS[field_name]['aliases'].index(alias)  # type: ignore
 
 
-def get_most_relevant(field_name, field_mappings):
-    candidates = sorted(field_mappings, key=lambda x: get_alias_index(field_name, x[1]))
+def get_most_relevant_json_field(field_name, json_field_to_alias):
+    if not json_field_to_alias:
+        return
```

Comment:
Also here check how it affects `match_for_incidents` that you might return None as a matching (perhaps filter none values at `match_for_incidents`)

Reply:
added a condition to verify non null there